apps/expo/src/app/(tabs)/(feeds,search,notifications,self)/search/index.tsx

```diff
@@ -277,11 +283,11 @@ const Suggestions = () => {
                                   {tag.name}
                                 </Text>{" "}
                                 <Text className="font-normal text-neutral-400 dark:text-neutral-500">
-                                  #{tag.tag}
+                                  <Trans>#{tag.tag}</Trans>
```

Comment:
Sorry to jump in unasked, just to check, does this need to be marked for translation? Is it for languages where the hashtag symbol appears after the tag, something like that? 🤔

Reply:
Yeah I thought there was a small chance a translator might wanna add some additional context here